hathor/manager.py

```diff
@@ -592,7 +615,17 @@ def _initialize_components_new(self) -> None:
         # XXX: last step before actually starting is updating the last started at timestamps
         self.tx_storage.update_last_started_at(started_at)
         self.state = self.NodeState.READY
-        self.log.info('ready')
+
+        t1 = time.time()
+        tdt = LogDuration(t1 - t0)
+
+        environment_info = self.environment_info.as_dict() if self.environment_info else {}
+
+        vertex_count = self.tx_storage.get_tx_count() + self.tx_storage.get_block_count()
+
+        # Changing the field names in this log could impact log collectors that parse them
+        self.log.info('ready', vertex_count=vertex_count,
```

Comment:
Note that I'm using here a new nomenclature `vertex_count` for the count of `txs` + `blocks`.

If it's better to use some other nomenclature (like `tx_count`), let me know.